src/TingBundle/Cache/MetadataWarmer.php

```diff
@@ -46,6 +46,11 @@ class MetadataWarmer implements CacheWarmerInterface
      */
     protected $repositories;
 
+    /**
+     * @var string
+     */
+    private $cacheFile;
```

Comment:
As it was not declared previously but initialized in constructor, it was a dynamic property with public visibility, so maybe private is a little too strict as it is a BC break, but I wonder if it matters?

Reply:
To avoid BC and keep coherence, I would declare it as `protected` like other properties of this class